packages/machine-extractor/src/__tests__/parseResult.test.ts

```diff
@@ -160,30 +160,21 @@ describe('MachineParseResult', () => {
     expect(config.states!.a.entry).toMatchInlineSnapshot(`
       {
         "type": "xstate.custom",
```

Comment:
We should have some way to represent anonymous actions... not sure what it would be yet

Reply:
What's wrong with representing anonymous actions as `xstate.custom`?